build.yml

```diff
@@ -180,10 +180,32 @@ docker-targets:
       xfonts-base
       zlib1g
 
-  centos8:
-    source: docker/Dockerfile.centos
+  almalinux9:
+    source: docker/Dockerfile.almalinux
+    args:
+      from: almalinux:9
+    output: rpm
+    matrix: ['x86_64', 'aarch64', 'ppc64le']
+    depend: >
+      ca-certificates
+      fontconfig
+      freetype
+      glibc
+      libjpeg
+      libpng
+      libstdc++
+      libX11
+      libXext
+      libXrender
+      openssl
```

Comment:
Shouldn't this be `compat-openssl11` as per #119?

Reply:
I think `compat-openssl11` is only needed when you try to install the existing RHEL 8 package on RHEL 9. When there is a native RHEL 9 package it should be linked against the correct OpenSSL version I guess, but I need to test this.

Reply:
I'm not sure it's going to work with OpenSSL 3.0, but please try and let me know if the build works.

Reply:
That's because it's using Qt5.